clang/lib/3C/ConstraintVariables.cpp

```diff
@@ -1697,19 +1816,32 @@ Atom *PointerVariableConstraint::getAtom(unsigned AtomIdx, Constraints &CS) {
   return nullptr;
 }
 
+bool PointerVariableConstraint::isFullyChecked(const EnvironmentMap &E) const {
+  for (const Atom *VA : Vars) {
+    const Atom *CA = getSolution(VA, E);
+    if (isa<WildAtom>(CA))
+      return false;
+  }
+  //if (FV)
+  //  return FV->isFullyChecked(E);
```

Comment:
Interesting that this is commented out. Why?

Reply:
I guess the point is that "fully checked" means the current data/pointer, not the thing it points to. Maybe the name of the function should be adjusted to make clear what it's doing? (I earlier had questions about it.)